index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+  <div class="container">
+    <header>
+      <nav class="navbar">
+        <div class="log">My Logo</div>
```

Comment:
- [ ] As requested by the previous reviewer, kindly replace the `div` tag with an `a` tag to make the logo a link to the home page 👍 
- [ ] As requested by the previous reviewer, kindly try to follow the Figma design as much as possible. Please update the `font-family` of the website to the one provided in the Figma design.